testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Standard.java

```diff
@@ -182,10 +181,16 @@ private static void enablePrestoJavaDebugger(DockerContainer container, String c
             container.withCopyFileToContainer(forHostPath(script), "/docker/presto-init.d/enable-java-debugger.sh");
 
             // expose debug port unconditionally when debug is enabled
-            exposePort(container, debugPort);
+            exposeFixedPort(container, debugPort);
         }
         catch (IOException e) {
             throw new UncheckedIOException(e);
         }
     }
+
+    private static void exposeFixedPort(DockerContainer container, int port)
```

Comment:
This will be needed in every place we handle `debug`. see eg TODO in the Hadoop.

Maybe we can keep the original method, just with a more fancy name?

Reply:
I was thinking about that but I'm worried that it will be misused.

Reply:
`exposeDebugPort` or `unsafelyExposePort` (plus some comment why it's not suitable in general case)